packages/paste-core/components/checkbox/__tests__/checkboxdisclaimer.test.tsx

```diff
@@ -119,16 +117,13 @@ describe('Customization', () => {
 
 describe('Accessibility', () => {
   it('Should have no accessibility violations', async () => {
-    const container = document.createElement('div');
     const errorText = 'This is the error text.';
-    document.body.append(container);
-    render(
+    const {container} = render(
       <CheckboxDisclaimer {...defaultProps} errorText={errorText}>
         foo
-      </CheckboxDisclaimer>,
-      container
+      </CheckboxDisclaimer>
     );
-    const results = await axe(document.body);
+    const results = await axe(container);
```

Comment:
Just consistency here. We use RTL to run axe tests everywhere else now, using react DOM was an old way.